react/src/components/BAIMenu.tsx

```diff
@@ -1,14 +1,47 @@
-import { ConfigProvider, Menu, MenuProps } from 'antd';
+import { ConfigProvider, Menu, MenuProps, theme } from 'antd';
+import { createStyles } from 'antd-style';
+import classNames from 'classnames';
 import React from 'react';
 
-const BAIMenu: React.FC<MenuProps> = ({ ...props }) => {
+const useStyle = createStyles(({ css, token }) => ({
+  baiMenu: css`
+    &.expanded ul.ant-menu-item-group-list li.ant-menu-item {
+      padding-inline: ${token.padding}px !important;
+    }
+
+    &.expanded li div.ant-menu-item-group-title {
+      padding-top: ${token.paddingMD}px;
+      padding-left: ${token.paddingXL}px;
+    }
+
+    &.expanded li div.ant-menu-item-group-title div {
+      border-bottom-width: 0 !important;
+    }
+    li div.ant-menu-item-group-title div span {
+      font-weight: 500;
+    }
+  `,
+}));
+
+interface BAIMenuProps extends MenuProps {
+  collapsed?: boolean;
+}
+
+const BAIMenu: React.FC<BAIMenuProps> = ({ collapsed, ...props }) => {
+  const { styles } = useStyle();
+  const { token } = theme.useToken();
+  const colorPrimaryWithAlpha = `rgba(${parseInt(token.colorPrimary.slice(1, 3), 16)}, ${parseInt(token.colorPrimary.slice(3, 5), 16)}, ${parseInt(token.colorPrimary.slice(5, 7), 16)}, 0.15)`;
+
   return (
     <ConfigProvider
       theme={{
         components: {
           Menu: {
-            itemBorderRadius: 2,
-            itemMarginInline: 0,
+            itemBorderRadius: 20,
+            itemMarginInline: 16,
```

Comment:
Is it okay if it's not managed by a token?

Reply:
Thank you for your question. The Menu component's `itemBorderRadius` and `itemMarginInline` are specifically tailored for the WebUI NEO Menuitem design, intended for use within the Sider menu. As this component focuses on a specific visual style rather than being part of a broader layout system, these values aren't necessarily managed as tokens.

Reply:
I'll update this part also. ;)